q2_feature_table/tests/test_merge.py

```diff
@@ -62,7 +63,8 @@ def test_invalid_overlapping_feature_ids(self):
         t2 = Table(np.array([[0, 2, 6], [2, 2, 4]]),
                    ['O1', 'O3'],
                    ['S4', 'S5', 'S6'])
-        with self.assertRaisesRegex(ValueError, 'features are present'):
+        with self.assertRaisesRegex(biom.exception.DisjointIDError,
```

Comment:
Would it make sense to trap the exception raised by `biom` in the method, that way we can maintain the same error message? I think it might be nice to maintain the same error message that users have already been exposed to - what do you think?

Reply:
Okay, trapping. It does obfuscate the traceback, unless we pull out the st but I'm not sure if the increased complexity of the error handling is sufficiently beneficial.